be/compendium/compendium/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('data/', include('db.urls')),
```

Comment:
add the additional route here `path('dailyReports/', DailyReportListAPIView.as_views())`
Note: `DailyReportListAPIView` needs to be imported first